3_javascript/javascript1/week1/script.js

```diff
@@ -0,0 +1,131 @@
+
+let yearOfBirth = 1982;
+let yearFuture = 2022;
+let age = yearFuture - yearOfBirth;
+console.log(`you will be ${age} years old in ${yearFuture}`);
+
```

Comment:
👍 Nice and clean implementation. The logic is spot on, and the code readability is good.
💡 I see that yearOfBirth and yearFuture are not being reassigned. Consider using const instead of let for these variables to signify they are constants.

Reply:
I have changed the 2 lets to const.